### DISCUSSION TITLE
用CPU 146k推理，卡住了。

### DISCUSSION THREAD

**Discussion by kbb522** [PaddlePaddle/PaddleOCR#17088] [Q&A]
环境：
Python 3.11.14
pip 25.2 from C:\Users\jbb\miniconda3\envs\paddle-orc\Lib\site-packages\pip (python 3.11)
64bit
AMD64

安装：
CPU 版的 PaddlePaddle
python -m pip install paddlepaddle==3.2.1 -i https://www.paddlepaddle.org.cn/packages/stable/cpu/

代码：
<img width="635" height="765" alt="ScreenShot_2025-11-14_140814_334" src="https://github.com/user-attachments/assets/8a39d073-d150-4263-b800-84aba1d15c29" />

日志如下：
C:\Users\jbb\miniconda3\envs\paddle-orc\python.exe C:\Users\jbb\PycharmProjects\ocr-test\PaddleOCRVL.py 
Creating model: ('PP-DocLayoutV2', None)
Model files already exist. Using cached files. To redownload, please delete the directory manually: `C:\Users\jbb\.paddlex\official_models\PP-DocLayoutV2`.
��Ϣ: ���ṩ��ģʽ�޷��ҵ��ļ���
C:\Users\jbb\miniconda3\envs\paddle-orc\Lib\site-packages\paddle\utils\cpp_extension\extension_utils.py:718: UserWarning: No ccache found. Please be aware that recompiling all source files may be required. You can download and install ccache from: https://github.com/ccache/ccache/blob/master/doc/INSTALL.md
  warnings.warn(warning_message)
Creating model: ('PaddleOCR-VL-0.9B', None)
Model files already exist. Using cached files. To redownload, please delete the directory manually: `C:\Users\jbb\.paddlex\official_models\PaddleOCR-VL`.
Loading configuration file C:\Users\jbb\.paddlex\official_models\PaddleOCR-VL\config.json
Loading weights file C:\Users\jbb\.paddlex\official_models\PaddleOCR-VL\model.safetensors
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
use GQA - num_heads: 16- num_key_value_heads: 2
C:\Users\jbb\miniconda3\envs\paddle-orc\Lib\site-packages\paddle\utils\decorator_utils.py:420: Warning: 
Non compatible API. Please refer to https://www.paddlepaddle.org.cn/documentation/docs/en/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.split.html first.
  warnings.warn(
Loaded weights file from disk, setting weights to model.
All model checkpoint weights were used when initializing PaddleOCRVLForConditionalGeneration.

All the weights of PaddleOCRVLForConditionalGeneration were initialized from the model checkpoint at C:\Users\jbb\.paddlex\official_models\PaddleOCR-VL.
If your task is similar to the task the model of the checkpoint was trained on, you can already use PaddleOCRVLForConditionalGeneration for predictions without further training.
Loading configuration file C:\Users\jbb\.paddlex\official_models\PaddleOCR-VL\generation_config.json
Currently, the PaddleOCR-VL-0.9B local model only supports batch size of 1. The batch size will be updated to 1.
C:\Users\jbb\miniconda3\envs\paddle-orc\Lib\site-packages\paddle\tensor\creation.py:1105: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach(), rather than paddle.to_tensor(sourceTensor).
  return tensor(
C:\Users\jbb\miniconda3\envs\paddle-orc\Lib\site-packages\paddle\utils\decorator_utils.py:420: Warning: 
Non compatible API. Please refer to https://www.paddlepaddle.org.cn/documentation/docs/en/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.max.html first.
  warnings.warn(

**Comment by kbb522** [comment]
而且，计算都给了小核心。
<img width="686" height="478" alt="ScreenShot_2025-11-14_144509_517" src="https://github.com/user-attachments/assets/294e6769-37f8-4c5e-9acb-97d1efe6cee6" />
